src/core/github.js

```diff
@@ -41,7 +53,7 @@ export async function run(conf) {
     const msg =
       "Config option `[github](https://github.com/w3c/respec/wiki/github)` " +
       "is missing property `repoURL`.";
-    pub("error", msg);
+    rejectGithubPromise(new Error(msg));
```

Comment:
Might be cleaner to just pass `msg` and create `Error` inside the function.

Reply:
Good idea.